### PR TITLE
added expandable heading

### PR DESCRIPTION
**Pull request by samMcCarthyBJSS**
Story AB#7461
Task AB#7691

### CODE REVIEW

File: app/components/general/components/bc-table/macro.njk [hunk 1]
```diff
@@ -7,10 +7,19 @@
 {% macro bcTable(params) %}
   <div data-test-id="table">
     {% if params.columnInfo %}
-      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 nhsuk-u-padding-bottom-3 bc-t-border-bottom--grey-3">
+      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 bc-t-border-bottom--grey-3">
         {% for column in params.columnInfo %}
           <div data-test-id="column-heading-{{ loop.index0 }}" class="nhsuk-u-padding-1 nhsuk-u-font-weight-bold {{ params.columnClass }}">{{ column.data }}</div>
         {% endfor %}
+        {% for column in params.columnInfo %}
```
Comment (alibombaye) [inline app/components/general/components/bc-table/macro.njk:14]: we got the for loop in there twice? can we just add the if clause to the one above?

File: app/components/general/components/bc-table/macro.njk [hunk 2]
```diff
@@ -7,10 +7,19 @@
 {% macro bcTable(params) %}
   <div data-test-id="table">
     {% if params.columnInfo %}
-      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 nhsuk-u-padding-bottom-3 bc-t-border-bottom--grey-3">
+      <div data-test-id="table-headings" class="nhsuk-u-font-size-19 nhsuk-grid-row nhsuk-u-margin-1 bc-t-border-bottom--grey-3">
```
Comment (kat1906) [inline app/components/general/components/bc-table/macro.njk:10]: @alibombaye pls can you check in admin that removing this padding bottom doesn't affect the tables in there

File: app/components/general/components/bc-table/macro.njk (hunk 1 above)
Comment (samMcCarthyBJSS) [reply app/components/general/components/bc-table/macro.njk:14]: makes it go weird when there's not the extra for loop in 
image

Comment (kat1906) [reply app/components/general/components/bc-table/macro.njk:14]: it is effectively another row in the headings

File: app/components/general/components/bc-table/macro.njk (hunk 2 above)
Comment (alibombaye) [reply app/components/general/components/bc-table/macro.njk:10]: ok I'll npm link it and see what happens

File: app/components/general/components/bc-table/macro.njk (hunk 1 above)
Comment (alibombaye) [reply app/components/general/components/bc-table/macro.njk:14]: mmmm doesn't seem right the one for the normal data row doesn't do that

File: app/components/general/components/bc-table/macro.njk (hunk 2 above)
Comment (alibombaye) [reply app/components/general/components/bc-table/macro.njk:10]: it looks fine as is

**Pull request merged**